### PR TITLE
Med 21 create portal window

### PR DESCRIPTION
**Pull request by pavlo-serhiichuk**

### CODE REVIEW

File: src/components/Portal/Portal.component.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {Form, Input, Dialog, Submit, Title, Wrapper, Avatar, Consultant, MessageWrap, Message} from "./Portal.style";
+import {AiOutlineSend, SiDialogflow} from "react-icons/all";
+
+const Portal = () => {
+
+    const messageArray = "How can I help you, Sapiens? I'm a big professional in different areas! Know everything about human nature. ".split(' ')
+    const message = messageArray.map(el => <Message>{el}</Message>)
+    console.log()
```
Comment (SVladikO) [inline src/components/Portal/Portal.component.js]: delete

File: src/components/Portal/Portal.component.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <Wrapper>
+            <Title> <SiDialogflow/>Free support</Title>
+            <Dialog>
+                <Consultant>
+                    <Avatar />
+                    <MessageWrap>{message}</MessageWrap>
+                </Consultant>
+            </Dialog>
+            <Form>
+                <Input type="text"/>
```
Comment (SVladikO) [inline src/components/Portal/Portal.component.js]: what's default type for input ?

File: src/components/Portal/Portal.component.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                <Submit type="submit"><AiOutlineSend size={35}/></Submit>
```
Comment (SVladikO) [inline src/components/Portal/Portal.component.js]: SubmitButton ?

File: src/components/Portal/Portal.component.js (hunk 1 above)
Comment (pavlo-serhiichuk) [reply src/components/Portal/Portal.component.js]: fixed

File: src/components/Portal/Portal.component.js (hunk 2 above)
Comment (pavlo-serhiichuk) [reply src/components/Portal/Portal.component.js]: fixed

File: src/components/Portal/Portal.component.js (hunk 3 above)
Comment (pavlo-serhiichuk) [reply src/components/Portal/Portal.component.js]: fixed

**Pull request merged**